bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/config/DimmerConfig.java

```diff
@@ -22,6 +22,7 @@
  */
 public class DimmerConfig {
     private int zoneNumber;
+    public int system = 0;
```

Comment:
private?

Reply:
The latest guidance on these config classes is to make the fields public. Probably in the interest of brevity, since they are effectively just DTOs. But they should at least be consistent. I'll go through and change them all to one or the other.